src/linker/Linker/Driver.cs

```diff
@@ -821,7 +821,11 @@ string GetNextStringValue ()
 		protected virtual LinkContext GetDefaultContext (Pipeline pipeline)
 		{
 			LinkContext context = new LinkContext (pipeline) {
+#if FEATURE_ILLINK
+				CoreAction = AssemblyAction.Link,
```

Comment:
Please tweak the documentation at https://github.com/sbomer/linker/blob/illinkDefaults/src/linker/Linker/Driver.cs#L869